src/main/java/edu/ksu/canvas/interfaces/SubmissionWriter.java

```diff
@@ -29,4 +31,13 @@ public interface SubmissionWriter extends CanvasWriter<Submission, SubmissionWri
      * @throws             IOException If there is an error talking to Canvas
      */
     public Optional<Progress> gradeMultipleSubmissionsByCourse(MultipleSubmissionsOptions options) throws IOException;
+
+    /**
+     * Submit an assignment on behalf of a student.
```

Comment:
While it can submit on behalf of a student, it only does that if you set the optional `user_id`